tachyon/math/elliptic_curves/short_weierstrass/affine_point.h

```diff
@@ -61,6 +62,13 @@ class AffinePoint<_Curve, std::enable_if_t<_Curve::kIsSWCurve>> final
     return ret;
   }
 
+  constexpr static std::optional<AffinePoint> CreateFromX(const BaseField& x,
+                                                          bool pick_bigger) {
+    AffinePoint point;
+    if (!Curve::GetPointFromX(x, pick_bigger, &point)) return std::nullopt;
```

Comment:
Typically, when selecting a `Point`, one cannot determine whether their y-value is large or small (requiring additional computation), so in Ethereum signatures, the y-value is chosen based on "odd/even." Don't you think we need to address this?